src/projectscene/qml/Audacity/ProjectScene/clipsview/TracksClipsView.qml

```diff
@@ -29,12 +29,13 @@ Rectangle {
     ProjectPropertiesModel {
         id: project
 
-        onCaptureThumbnail: function captureThumbnail() {
+        onCaptureThumbnail: function captureThumbnail(thumbnailUrl) {
             // hide playCursor for the time grabbing image
             playCursor.visible = false
             content.grabToImage(function(result) {
                 playCursor.visible = true
-                result.saveToFile(project.thumbnailUrl)
```

Comment:
better to remove `thumbnailUrl` property from ProjectPropertiesModel

Reply:
Done.